<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeObject</class>
 <widget class="QWidget" name="HomeObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>591</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="singleModeButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Single Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="groupModeButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="numPlayersLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of players:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numPlayersBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
